packages/bpk-docs/src/pages/BannerAlertsPage/BannerAlertsPage.js

```diff
@@ -84,6 +84,81 @@ BpkBannerDismissable.defaultProps = {
   type: null,
 };
 
+// eslint-disable-next-line react/no-multi-comp
+class BpkBannerAlertDismissDemo extends Component {
+  constructor() {
+    super();
+
+    this.newBannerAlerts = [
+      {
+        show: true,
+        message: 'Neutral alert with dismiss option.',
+        type: ALERT_TYPES.NEUTRAL,
+      },
+      {
+        show: true,
+        message: 'Successful alert with dismiss option.',
+        type: ALERT_TYPES.SUCCESS,
+      },
+      {
+        show: true,
+        message: 'Warn alert with dismiss option.',
+        type: ALERT_TYPES.WARN,
+      },
+      {
+        show: true,
+        message: 'Error alert with dismiss option.',
+        type: ALERT_TYPES.ERROR,
+      },
+    ];
+
+    this.reset = this.reset.bind(this);
+    this.setDismissed = this.setDismissed.bind(this);
+
+    this.state = {
+      bannerAlerts: JSON.parse(JSON.stringify(this.newBannerAlerts)),
+    };
+  }
+
+  setDismissed(index) {
+    const updatedBannerAlerts = JSON.parse(JSON.stringify(this.state.bannerAlerts));
+    updatedBannerAlerts[index].show = false;
+    this.setState({
+      bannerAlerts: updatedBannerAlerts,
+    });
+  }
+
+  reset() {
+    this.setState({
+      bannerAlerts: JSON.parse(JSON.stringify(this.newBannerAlerts)),
```

Comment:
How come you're stringifying it and then parsing it?

Reply:
Javascript doesn't have a built in clone option. This is the most performant way to achieve the same without relying on 3rd party stuff!

Reply:
Yeah fair enough - I googled it and came to the same thing! https://stackoverflow.com/a/23481096/1011161